src/MICore/Transports/ITransport.cs

```diff
@@ -16,6 +16,13 @@ public interface ITransport
         void Send(string cmd);
         void Close();
         bool IsClosed { get; }
+
+        /// <summary>
+        /// Process ID of the debugger process (clrdbg/lldb/gdb).
```

Comment:
I would also suggest adding -
This value is only valid when using local launch options. When using non-local options this may throw (TcpTransport) or provide bogus data (PipeTransport).